packages/service-library/tests/fastapi/test_http_client.py

```diff
@@ -98,5 +99,22 @@ class MyClientApi(BaseHttpApi, AppStateMixin):
         assert await api.ping()
         assert await api.is_healhy()
 
+        alive = await api.check_liveness()
+        assert bool(alive)
+        assert isinstance(alive, IsResponsive)
+        assert alive.elapsed.total_seconds() < 1
+
     # shutdown event
     assert api.client.is_closed
+
+
+async def test_to_curl_command(mock_server_api: respx.MockRouter, base_url: str):
+
+    mock_server_api.post(path__startswith="/foo").respond(status.HTTP_200_OK)
```

Comment:
can you also add a case where you test the `GET` method. I have some doubts it would work. I'm not sure that curl likes it when you passa a body in a get request

Reply:
I added a test to convince you ;-)